src/main/java/ch/heigvd/pro/b04/votes/Vote.java

```diff
@@ -0,0 +1,27 @@
+package ch.heigvd.pro.b04.votes;
+
+import ch.heigvd.pro.b04.answers.Answer;
+import ch.heigvd.pro.b04.participants.Participant;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@NoArgsConstructor
```

Comment:
We probably want a Builder as well, to be able to test it correctly and easily

Reply:
ok...